src/nav/nav_model/nav_model_rings_base.py

```
@@ -0,0 +1,207 @@
+        label_vert_gap = getattr(rings_config, 'label_vert_gap',
+                                 bodies_config.label_vert_gap)
+
+        if not edge_info_list:
+            return annotations
+
+        # Get gap configuration from bodies config (for label placement)
+        label_mask_enlarge = getattr(rings_config, 'label_mask_enlarge',
+                                     bodies_config.label_mask_enlarge)
+
+        # Create annotations for each edge
+        for edge_mask, label_text, edge_type in edge_info_list:
```

Ruff (0.14.10)

Loop control variable `edge_type` not used within loop body

Rename unused `edge_type` to `_edge_type`

(B007)